requirements/base.txt

```diff
@@ -12,7 +12,7 @@ git+https://github.com/edx/edx-submissions.git@1.1.4#egg=edx-submissions==1.1.4
 boto>=2.32.1,<3.0.0
 celery==3.1.18
 defusedxml==0.4.1
-django>=1.8.17,<1.9
+django<1.9a0 # 'a' resolves known bug on gemnasium
```

Comment:
- I would add a link to docs about the bug.  I've got questions about it, but maybe docs would answer.
- Is this an artifact of switching from `<1.9` to `<1.9.0`?  Did we already test with the former?
- Do we need to test an edx-platform build against this?